src/main/java/org/javarosa/xml/ElementParser.java

```diff
@@ -74,6 +76,20 @@ public static KXmlParser instantiateParser(InputStream stream) throws IOExceptio
         }
     }
 
+    /**
+     * Prepares a parser that will be used by the element parser, configuring relevant
+     * parameters and setting it to the appropriate point in the document.
+     *
+     * @param stream A input stream which is reading the XML content
+     *               of the document.
+     * @throws IOException If the stream cannot be read for any reason
+     *                     other than invalid XML Structures.
+     */
+    public static KXmlParser instantiateParser(InputStream stream) throws IOException {
```

Comment:
sorry did not see this overload before my comment... 👍 delete the comment anyway.